autogpt_platform/frontend/src/lib/autogpt-server-api/types.ts

```diff
@@ -879,6 +879,36 @@ export type AdminPendingSubmissionsRequest = {
   page_size: number;
 };
 
+export enum CreditTransactionType {
+  TOP_UP = "TOP_UP",
+  USAGE = "USAGE",
+  GRANT = "GRANT",
+  REFUND = "REFUND",
+  CARD_CHECK = "CARD_CHECK",
+}
+
+export type UserBalanceHistory = {
```

Comment:
Can we unify TransactionHistory instead? they should be care about the same information.